docs/kfs_example.md

```diff
@@ -0,0 +1,141 @@
+# Preview support of KServe gRPC API{#ovms_kserve}
+
+OpenVINO Model Server 2022.2 release introduce preview of support for KServe gRPC API.
+
+Inference does support putting tensor buffers both in `ModelInferRequest` `InferTensorContents` and `raw_input_contents`. There is no support for BF16 data type and there is no support for using FP16 in `InferTensorContents`.
+In case of sending raw images BYTES data type is used. When using raw images `InferTensorContents`.
```

Comment:
This is some logical mistake here.

Reply:
I think it could be explained here how input data should be passed in an array format and encoded jpeg content. Also the limitations in the encoded format (only in InferTensorContents for now)